src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateGlobalUsings.targets

```diff
@@ -0,0 +1,64 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.GenerateGlobalUsings.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    ============================================================
+                                     GenerateGlobalUsings
+
+    Generates global using source to intermediate directory
+    ============================================================
+    -->
+  <PropertyGroup>
+    <GeneratedGlobalUsingsFile Condition="'$(GeneratedGlobalUsingsFile)' ==''">$(IntermediateOutputPath)$(MSBuildProjectName).GlobalUsings$(DefaultLanguageSourceExtension)</GeneratedGlobalUsingsFile>
+    <GenerateGlobalUsings Condition="'$(GenerateGlobalUsings)' == '' and '$(LangVersion)' == 'preview'">true</GenerateGlobalUsings>
```

Comment:
Currently the logic defaults to true when:
1. Is a C# project (condition on import in Microsoft.NET.Sdk.targets)
2. LangVersion is set to preview (we can probably remove this in preview7, I just need to test with the latest SDK)
3. It's not explicitly disabled by the user
4. The TFM is net6.0 or higher (condition on the ItemGroup in this file and in Microsoft.NET.Sdk.Web.ProjectSystem.targets)

We should consider the alternative where this is an opt in feature and our `web api` and `empty web` templates sets `GenerateGlobalUsings` to true.

Reply:
The plan is for this to be enabled by default.